packages/generators/app/lib/utils/usage.js

```diff
@@ -107,14 +118,12 @@ function trackError({ scope, error }) {
 }
 
 function trackUsage({ event, scope, error }) {
```

Comment:
I think the `trackUsage` function should also benefit from the API change introduced to the telemetry sending function in the core. I'm talking about the possibility of also passing event & user properties.

The same could be said in the `useTracking` hook in the frontend.

Reply:
Not sure what exactly you mean here? `trackUsage` has been changed to correspond to the API change. In `trackUsage` `getProperties` returns the correct new format of properties, but since in the generators all that info is static, there is no need to pass anything else to it. And in `useTracking` everything has been changed to correspond to the same way the backend sender does it. Am I missing something? 😅

Reply:
Both functions are now indeed compatible with the new format in the way they send the data.
But their input interface is not. The consumers of those functions do not have the possibility to categorize properties.

Again as I said, this requires too many changes most likely, so we can ignore that from now but let's keep it in mind. We would want to avoid discrepancies throughout the code on how the telemetry calls are made.